pallets/pallet-dyn-filter/src/setting.rs

```diff
@@ -0,0 +1,28 @@
+// KILT Blockchain – https://botlabs.org
+// Copyright (C) 2019-2022 BOTLabs GmbH
+
+// The KILT Blockchain is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+
+// The KILT Blockchain is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+// If you feel like getting in touch with us, you can do so at info@botlabs.org
+
+use codec::{Decode, Encode, MaxEncodedLen};
+use frame_support::RuntimeDebug;
+use scale_info::TypeInfo;
+
+#[derive(Copy, Clone, Encode, Decode, TypeInfo, MaxEncodedLen, Default, RuntimeDebug, PartialEq, Eq)]
+pub struct FilterSettings {
+	pub transfer: bool,
```

Comment:
I would rename the fields of this type. What does it mean for the `transfer` field to be `true`? Is it filtered or is it allowed? Same for the other ones of course.

Reply:
043a6f285df9f1149fb462d905bdce0527b06cf3